README.md

```diff
@@ -59,6 +59,11 @@ Apply all outstanding migrations:
 liquibase update --changelog-file ./etor/databaseMigrations/root.yml --url jdbc:postgresql://localhost:5433/intermediary --username intermediary --password 'changeIT!' --label-filter '!azure'
 ```
 
+If running in Windows, use double quotes instead:
```

Comment:
If OSX and linux work with double quotes, we can just replace the single quotes for double. I can try this in OSX

Reply:
Hmm I tried it and it doesn't work, so nevermind